pages/index.vue

```diff
@@ -47,6 +47,13 @@
             </div>
         </div>
 
+        <i18n-page-wrapper>
+            <div class="bulletin-section">
+                <core-h2 :title="$t('bulletinList')" bulletin-color></core-h2>
+                <bulletin-card-collection></bulletin-card-collection>
+            </div>
+        </i18n-page-wrapper>
+
```

Comment:
**issue (blocking):** this section should be placed between the 
`Sponsors` section and the `PyCon Taiwan` section

<img width="743" alt="image" src="https://user-images.githubusercontent.com/18432820/174454379-3d16bb72-5744-45db-abed-1e23e08e6b16.png">

Reply:
Fixed